src/containerbuddy/main.go

```diff
@@ -111,6 +112,18 @@ func getCmd(args []string) *exec.Cmd {
 	}
 }
 
+func runCommand(commandString string) (int, error) {
```

Comment:
I feel like it makes more sense to move the string splitting into the `run`, and then we can eliminate `ServiceConfig.healthArgs` and `Backend.onChangeArgs` members too (and the splitting code in config.go).